composer.json

```diff
@@ -10,7 +10,10 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0"
+        "php": "^8.1"
```

Comment:
can you add psalm dependency back? removed it in the last commit in main branch, but it causes more issues than it solves.